File: iclusion-importer/src/main/java/com/hartwig/hmftools/iclusion/data/IclusionIndication.java
package com.hartwig.hmftools.iclusion.data;

import java.util.List;

import com.squareup.moshi.Json;

public class IclusionIndication {
    @Json(name = "id") public String id;
    @Json(name = "parent_id") public String parentId;
    @Json(name = "doid") public String doid;
    @Json(name = "doid2") public String doid2;
    @Json(name = "indication_name") public String indicationName;
    @Json(name = "indication_name_full") public String indicationNameFull;
    @Json(name = "node_ids") public List<String> nodeIds;
}

File: api-clients/src/main/kotlin/com/hartwig/hmftools/apiclients/iclusion/data/IclusionIndication.kt
package com.hartwig.hmftools.apiclients.iclusion.data

data class IclusionIndication(val id: String, val doid: String?, val doid2: String?, val indication_name: String,
                              val indication_name_full: String, val parent_id: String, val node_ids: List<String>) {

    val doidSet: Set<String> = listOfNotNull(doid, doid2).toSet()
}
